src/components/views/messages/MPollBody.tsx

```diff
@@ -132,16 +152,21 @@ export function findTopAnswer(
 export function isPollEnded(
     pollEvent: MatrixEvent,
     matrixClient: MatrixClient,
-    getRelationsForEvent?: (
-        eventId: string,
-        relationType: string,
-        eventType: string
-    ) => Relations,
+    getRelationsForEvent?: GetRelationsForEvent,
 ): boolean {
     if (!getRelationsForEvent) {
         return false;
     }
 
+    const pollEventId = pollEvent.getId();
+    if (!pollEventId) {
+        logger.warn(
+            "isPollEnded: Poll event must have event ID in order to determine whether it has ended " +
+            "- assuming poll has not ended",
+        );
+        return false;
```

Comment:
Is it better to fail open or closed here?

Reply:
To keep the current behaviour I would say `false`:
- no event id,  no relations → `false`
- without room `userCanRedact` should return `false` → filter for return value will be always `false` → `false`